### PR TITLE
Addresses #2119 NTR perivascular cell

### LINKED ISSUE #2119: [NTR] perivascular cell
This term was requested during the Breast atlas jam (08/2023) as a grouping term for vascular smooth muscle cell and pericyte.

**Preferred term label**
perivascular cell

**Synonym(s)** (add reference(s), please)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
A cell that is adjacent to a vessel. A perivascular cell plays a crucial roles in maintaining vascular function and tissue homeostasis. This cell regulates vessel integrity and flow dynamics.
[PMID:27889329](https://pubmed.ncbi.nlm.nih.gov/27889329)
[PMID: 24928499](https://pubmed.ncbi.nlm.nih.gov/24928499)

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
'native cell'

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
'adjacent to' some vasculature

**Your ORCID**
https://orcid.org/0000-0001-6677-8489

**Additional notes or concerns** 
Cell types that will be SubclassOf 'perivascular cell':
 - vascular associated smooth muscle cell
 - mural cell
 - perivascular macrophage

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #2119 NTR perivascular cell

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -31843,6 +31847,14 @@ AnnotationAssertion(rdfs:label obo:CL_4033053 "small bistratified retinal gangli
 SubClassOf(obo:CL_4033053 obo:CL_0003001)
 SubClassOf(obo:CL_4033053 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001792))
 
+# Class: obo:CL_4033054 (perivascular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24928499") Annotation(oboInOwl:hasDbXref "PMID:27889329") obo:IAO_0000115 obo:CL_4033054 "A cell that is adjacent to a vessel. A perivascular cell plays a crucial roles in maintaining vascular function and tissue homeostasis. This cell regulates vessel integrity and flow dynamics.")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31852]: A perivascular cell plays a crucial roles in -->
A perivascular cell plays a crucial **role** in
or
A perivascular cell **plays crucial** roles in

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
See inline comment.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -31843,6 +31847,14 @@ AnnotationAssertion(rdfs:label obo:CL_4033053 "small bistratified retinal gangli
 SubClassOf(obo:CL_4033053 obo:CL_0003001)
 SubClassOf(obo:CL_4033053 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001792))
 
+# Class: obo:CL_4033054 (perivascular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24928499") Annotation(oboInOwl:hasDbXref "PMID:27889329") obo:IAO_0000115 obo:CL_4033054 "A cell that is adjacent to a vessel. A perivascular cell plays a crucial role in maintaining vascular function and tissue homeostasis. This cell regulates vessel integrity and flow dynamics.")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31852]: Consider editing last sentence to: This cell **type** regulates vessel integrity and flow dynamics.

Will approve PR and leave this edit to your discretion.

**Review by bvarner-ebi — APPROVED** [review]

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:31852]: Thank you for the suggestion, I will add it now.

**Pull request merged**